crates/node/src/types/file.rs

```diff
@@ -18,6 +18,14 @@ impl<E> TaskVmFile<E> {
         &self.vm_file_path
     }
 }
+impl TaskVmFile<()> {
+    pub fn get_workspace_path(data_directory: &Path, tx_hash: Hash) -> PathBuf {
+        PathBuf::new()
+            .join(data_directory)
+            .join(&tx_hash.to_string())
```

Comment:
Yes I agree with you. I add the same idea but I didn't change because I wanted to keep things like they were.
I can add a directory in this PR or open a new PR to do the change?
I prefer data or files.

Reply:
>  open a new PR to do the change?

Open a new PR so it's clearer that way :+1:

> I prefer data or files.

Both work for me. Possibly prefix with "tx" so it's absolutely obvious what it's about. :+1: